### DISCUSSION TITLE
How to bind 2 or more browser flow

### DISCUSSION THREAD

**Discussion by alirf81** [keycloak/keycloak#14145] [Q&A]
For user convenience, I need to configure 2 or more custom browser flows.
So my idea is to have a kind of "primary" browser flow,
and put buttons on login page which the users can click and redirect to custom browser login flow.
Any idea of how I can implement this?

**Comment by jbman** [comment]
You can configure a client specific flow at client section "Authentication Flow Overrides". Your button can link to a page where a second client initiates authorization.

**Reply by alirf81** [reply]
Thank you @jbman 
Yeah, it would work for sure, but in my case, it's difficult to create new client, are there any other way?
Something like to define sub-flow and trigger it by specific URL?
